react/components/TableList.tsx

```diff
@@ -48,15 +48,22 @@ const TableListArea: FC = () => {
       <div style={{ padding: '0 4rem' }}>
         <DataView state={view}>
           <DataViewControls>
-            <Search
-              id="search"
-              placeholder={formatMessage(table.searchList)}
-              {...getInputProps()}
-            />
-            <Toolbar state={toolbar} aria-label="Toolbar Render Props">
+            <div style={{ marginTop: '5px' }}>
```

Comment:
Hello, this is just a suggestion! I can see that you are using the admin-ui in this screens. I think that in this case you could use the `tag.div` instead of the `div` alone. With that you can apply styles to that div in the same way you apply to the admin-ui components using `csx` prop.

You can find an example in the admin-ui [docs](https://admin-ui.vercel.app/teams/developing/tag)

```
<tag.div
  csx={{
    bg: 'muted',
    color: 'base',
    border: 'default',
    borderRadius: 'default',
    padding: 4,
  }}
>
  admin-ui-powered div
</tag.div>
```

Reply:
Thanks for the suggestoin, I will change here!!